### ISSUE TITLE
Can't reproduce the classification results

### ISSUE THREAD

**Issue by PhoneSix** [nikitakaraevv/pointnet#25]
Hi, @nikitakaraevv Thanks a lot for your code! But I can't reproduce the classification results. Could you kindly give some suggestions? Thanks a lot!

1.

I download the save.pth and run the test code in PoinetNetClass.ipyb, but get really different results with the reported, especially for desk and night stand, as shown in the following table.
|  method  | Bathtub | Bed | Chair | Desk | Dresser | Monitor | Night stand | Sofa | Table | Toilet |  Avg |
|:--------:|:-------:|:---:|:-----:|:----:|:-------:|:-------:|:-----------:|:----:|:-----:|:------:|:----:|
| reported |   93.4  |  92 |  97.2 | 81.5 |    71   |   89.4  |      56     | 86.9 |  93.4 |  95.9  |  82  |
| save.pth |    82   |  97 |  100  |  40  |    49   |    73   |      90     |  95  |   93  |   83   | 80.2 |

2.
I retrain the model with default hyper parameters. Except for comment one line of code, since it prevents me from setting num_workers in dataloader as 4.
`# random.seed = 42`
The results is still different with the reported rsults, as shown in the following table.
|   method   | Bathtub | Bed | Chair | Desk | Dresser | Monitor | Night stand | Sofa | Table | Toilet |  Avg |
|:----------:|:-------:|:---:|:-----:|:----:|:-------:|:-------:|:-----------:|:----:|:-----:|:------:|:----:|
|  reported  |   93.4  |  92 |  97.2 | 81.5 |    71   |   89.4  |      56     | 86.9 |  93.4 |  95.9  |  82  |
|  save.pth  |    82   |  97 |  100  |  40  |    49   |    73   |      90     |  95  |   93  |   83   | 80.2 |
| reproduced |    80   |  98 |   93  |  38  |    52   |    92   |      88     |  86  |   98  |   88   | 81.3 |

3.
I check your training process in the PoinetNetClass.ipyb, the valid accuracy is still not stable near 15th epoch, around 4% gap between adjacent epochs. Any suggestions to make it stable?

**Comment by PhoneSix** [comment]
The results of my rerunned 'save.pth' is also different with the results in PoinetNetClass.ipyb.
